linkup/src/lib.rs

```diff
@@ -85,6 +85,20 @@ pub fn get_additional_headers(
         _ => {}
     }
 
+    let baggage = headers.get(HeaderName::Baggage);
+    let linkup_session = format!("linkup-session={}", session_name,);
+    match baggage {
+        Some(ts) if !ts.contains(&linkup_session) => {
+            let new_baggage = format!("{},{}", ts, linkup_session);
+            additional_headers.insert(HeaderName::Baggage, new_baggage);
+        }
+        None => {
+            let new_baggage = linkup_session;
+            additional_headers.insert(HeaderName::Baggage, new_baggage);
+        }
+        _ => {}
```

Comment:
**_question:_**
I don't think this is necessary. I guess `headers.get` returns an `Option` that can only be `Some` or `None` 🤔